projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -420,7 +420,6 @@ function wpcom_launchpad_get_task_definitions() {
 				return __( 'Enable subscribers modal', 'jetpack-mu-wpcom' );
 			},
 			'is_complete_callback' => 'wpcom_launchpad_is_task_option_completed',
-			'is_visible_callback'  => 'wpcom_launchpad_is_enable_subscribers_modal_visible',
```

Comment:
@daledupreez feel free to ignore review request, added you just because I was modifying launchpad files here.

Reply:
Thanks for the ping -- this removal makes sense to me if you want to relax the current restrictions.